internal/servers/worker/worker.go

```diff
@@ -26,6 +27,8 @@ type Worker struct {
 	baseCancel  context.CancelFunc
 	started     *ua.Bool
 
+	tickerWg *sync.WaitGroup
```

Comment:
Similarly here, this can probably be `sync.WaitGroup` and `Start` and `Shutdown` can be updated to remove the `new` and nil check.